gradle.properties

```diff
@@ -16,8 +16,8 @@
 jmhOutputPath=build/reports/jmh/human-readable-output.txt
 jmhJsonOutputPath=build/reports/jmh/results.json
 jmhIncludeRegex=.*
-systemProp.defaultFlinkVersions=1.20
-systemProp.knownFlinkVersions=1.18,1.19,1.20
+systemProp.defaultFlinkVersions=1.20,2.0
```

Comment:
Why 2 defaults?

Reply:
I thought it could be helpful for devs to have two defaults. Which version do you prefer to be active?

Reply:
typically, we only set the default to the newest version.